genny/auth/auth.go

```diff
@@ -0,0 +1,199 @@
+package auth
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"strings"
+
+	"github.com/gobuffalo/buffalo/meta"
+	"github.com/gobuffalo/flect"
+	"github.com/gobuffalo/genny"
+	"github.com/gobuffalo/genny/movinglater/gotools"
+	"github.com/gobuffalo/genny/movinglater/plushgen"
+	"github.com/gobuffalo/packr"
+	"github.com/gobuffalo/plush"
+	"github.com/pkg/errors"
+)
+
+// New actions/auth.go file configured to the specified providers.
+func New(args []string) (*genny.Generator, error) {
+	g := genny.New()
+
+	fields, extraFields := extractFields(args)
+
+	parts := append([]string{"db", "generate", "model"}, fields...)
+	g.Command(exec.Command("buffalo", parts...))
+
+	if err := g.Box(packr.NewBox("./templates")); err != nil {
```

Comment:
Due to a weird `packr@v1` bug, can you change the name of this box to "../auth/templates" instead?